pkg/executor/aggfuncs/spill_serialize_helper.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+import (
+	"github.com/pingcap/tidb/pkg/util/spill"
+)
+
+// SpillSerializeHelper can only be used for one aggregator function.
+// It may cause error if agg func1 and agg func2 use the same SpillSerializeHelper.
+type SpillSerializeHelper struct {
+	buf []byte
+}
+
+// NewSpillSerializeHelper creates a new SpillSerializeHelper
+func NewSpillSerializeHelper() *SpillSerializeHelper {
+	return &SpillSerializeHelper{
+		buf: make([]byte, 1024),
+	}
+}
```

Comment:
```suggestion
// serializeHelper can only be used for one aggregator function.
// It may cause error if agg func1 and agg func2 use the same serializeHelper.
type serializeHelper struct {
	buf []byte
}

// newSerializeHelper creates a new SpillSerializeHelper
func newSerializeHelper() *serializeHelper {
	return &serializeHelper{
		buf: make([]byte, 1024),
	}
}
```

Reply:
I will change the name, but we need the helper to be public as other package will access it.